src/app/participant-list/participant-list.component.ts

```diff
@@ -737,6 +738,9 @@ export class ParticipantListComponent implements OnInit {
       }
     );
   }
+  selectedColumnsNotEmpty(): boolean {
```

Comment:
I personally always prefer to follow convention about method which returns boolean value and use `is` prefix to give method quick understanding what it checks, or `has` if `is` does not fit.